HSTracker/UIs/Battlegrounds/BattlegroundsTierDetailsView.swift

```diff
@@ -58,10 +58,64 @@ class BattlegroundsTierDetailsView: NSStackView {
             let cardBar = CardBar.factory()
             cardBar.card = card
             return cardBar
-        }.sorted(by: {(a: CardBar, b: CardBar) -> Bool in
-            score(race: a.card?.race) > score(race: b.card?.race)
-        })
+        }
         
+        var cardBar = CardBar.factory()
+
+        let size = NSSize(width: cardBar.imageRect.width, height: cardBar.imageRect.height)
+        let blackImage = NSImage(color: NSColor(red: 35/255.0, green: 39/255.0, blue: 42/255.0, alpha: 1.0), size: size)
+
+        cardBar.playerName = "Neutral"
+        let race = Race(rawValue: "invalid")
+        cardBar.playerRace = race
+        cardBar.backgroundImage = blackImage
+        cardBars.append(cardBar)
+        for i in 0...availableRaces!.count-1 {
+            let race: String = availableRaces![i].rawValue
+            cardBar = CardBar.factory()
+            if availableRaces![i] == .mechanical {
+                cardBar.playerName = "Mech"
```

Comment:
That's purely cosmetic, right? To display "Mech" instead of "Mechanical"?

Reply:
Yes, it is the same thing that HDT does except it used EnumConverter and also took care of `pet` to `beast`. Just tried to keep it simple here.